lms/djangoapps/api_manager/users/tests.py

```diff
@@ -548,6 +548,13 @@ def test_user_detail_post_user_profile_added_updated(self):
         # Testing profile updating scenario.
         # Must be updated
 
+        data['first_name'] = "First Name"
+        data['last_name'] = "Surname"
```

Comment:
Nit: for consistency sake I'd use either "Name/Surname" or "First Name / Last Name" as values here.